test/test_datasets.py

```diff
@@ -45,6 +46,10 @@ def test_cmuarctic(self):
         data = CMUARCTIC(self.path)
         data[0]
 
+    def test_libritts(self):
```

Comment:
Can you make a separate test case? I see putting everything on the same `TestCase` is a bad pattern.

Also, since the quality of the audio file does not really matter, can you switch to generate all the test asset on-the-fly? (both audio and text) See the direction in https://github.com/pytorch/audio/issues/764.

Reply:
Thanks for the suggestion. I tried to make a separate test case and generate the audio and text on-the-fly. It is based on #792 .